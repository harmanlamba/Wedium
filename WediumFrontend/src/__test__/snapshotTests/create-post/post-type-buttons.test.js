import React from 'react';
import { shallow } from 'enzyme';

import PostTypeButtons from '../../../components/create-post/post-type-buttons';

it('snapshot postTypeButtons default load', () => {
  const setRouteLeaveHook = jest.fn();

  let wrapper = shallow(
    <PostTypeButtons.WrappedComponent
      params={{
        router: setRouteLeaveHook,
      }}
    />
  );

  expect(wrapper).toMatchSnapshot();
});
