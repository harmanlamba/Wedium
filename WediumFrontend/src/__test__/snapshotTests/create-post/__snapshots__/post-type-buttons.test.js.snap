// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot postTypeButtons default load 1`] = `
<PostTypeButtons
  classes={
    Object {
      "selected": "PostTypeButtons-selected-3",
      "toggleButton": "PostTypeButtons-toggleButton-2",
      "toggleGroup": "PostTypeButtons-toggleGroup-1",
    }
  }
  params={
    Object {
      "router": [MockFunction],
    }
  }
/>
`;
